core/controller/src/main/scala/org/apache/openwhisk/core/controller/AuthorizedRouteDispatcher.scala

```diff
@@ -64,6 +65,8 @@ trait BasicAuthorizedRouteProvider extends Directives {
   /** Route directives for API. The methods that are supported on entities. */
   protected lazy val entityOps = get
 
+  private val logger = new PrintStreamLogging()
```

Comment:
Please never use `PrintStreamLogging` in production code. It has severe impact on performance. We always use `AkkaLogging` there.

Reply:
Already applied AkkaLogging